caffe2/serialize/inline_container.cc

```diff
@@ -115,7 +124,7 @@ void PyTorchStreamReader::init() {
   // version check
   at::DataPtr version_ptr;
   size_t version_size;
-  std::tie(version_ptr, version_size) = getRecord("version");
+  std::tie(version_ptr, version_size) = getRecord(version_location_);
```

Comment:
I think the stream reader and writer should still handle the version number completely internally. After all, how can you specify a version location if you do not know the version of the package you are reading? It might be older. Instead, I feel like it can still do the exact same thing it did before: figure out the version of the package. The implementation just first checks for the new `.data/version` and if it is missing looks up `version`.

Reply:
sure that works for me